### PR TITLE
RF-2618 Bump pronto version

### PR DESCRIPTION
**Pull request by jbarber**
So we can use `proto --staged` to check issues locally in pre-commit
hooks

I changed @config => @commentator_config because it was colliding with
an instance variable introduced to Pronto::Runner

### CODE REVIEW

File: lib/pronto/commentator.rb [hunk 1]
```diff
@@ -44,7 +44,7 @@ def repo_path
     def config
       conf_file = File.join CONFIG_DIR, 'config.yml'
 
-      @config ||= YAML.load(File.read(conf_file))
+      @commentator_config ||= YAML.load(File.read(conf_file))
```
Comment (shosti) [inline lib/pronto/commentator.rb:47]: Why this change? (It's generally idiomatic to name a "memoized" ivar the same thing as its method...)

File: lib/pronto/commentator/version.rb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 module Pronto
   module CommentatorVersion
-    VERSION = '0.1.0'
+    VERSION = '0.1.1'
```
Comment (shosti) [inline lib/pronto/commentator/version.rb:4]: I'm not really an expert in semver, but technically wouldn't dependency changes be transitive? So a "minor" bump in the dependency would result in a "minor" bump here (i.e. to 0.2.0)?

**Review by shosti — APPROVED** [review]
Some bikeshedding, but looks fine.

File: lib/pronto/commentator.rb (hunk 1 above)
Comment (shosti) [reply lib/pronto/commentator.rb:47]: Oh nvm, just saw your comment, ignore ^.

File: lib/pronto/commentator/version.rb (hunk 2 above)
Comment (jbarber) [reply lib/pronto/commentator/version.rb:4]: http://semver.org/ says that for X.Y.Z, X + Y must change if we change our API behavior - AFAIK, we're not doing this, so we shouldn't change them.

But it also says if we're using the software in prod X should be 1. So... lol

> What should I do if I update my own dependencies without changing the public API?
> That would be considered compatible since it does not affect the public API. Software that explicitly depends on the same dependencies as your package should have their own dependency specifications and the author will notice any conflicts. Determining whether the change is a patch level or minor level modification depends on whether you updated your dependencies in order to fix a bug or introduce new functionality. I would usually expect additional code for the latter instance, in which case it’s obviously a minor level increment.

I think we can do either and it's fine? I'm more worried about how I test circlemator with this change tbh.

**Comment by shosti** [comment]
OK looks fine.
